flint.ui/src/components/GCBM/Configurations/Pools.vue

```diff
@@ -1,27 +1,39 @@
 <template>
   <div class="py-4 px-8 text-earth">
-    <a-typography-title><span class="font-normal text-earth"> Configure Pools </span></a-typography-title>
+    <a-typography-title
+      ><span class="font-normal text-earth">
+        Configure Pools
+        <a-input-search
+          v-model:value="searchValue"
+          placeholder="Search for a pool"
+          style="width: 250px; margin-left: 28rem; margin-top: 2px"
+          class="mb-6"
+        /> </span
+    ></a-typography-title>
     <a-typography-text>
-      <span class="text-earth mb-2 block"> Configure the starting points for each pool. </span>
+      <span class="text-earth mb-2 block" style="margin-bottom: 20px">
+        Configure the starting points for each pool.
+      </span>
     </a-typography-text>
-    <a-input-search v-model:value="searchValue" placeholder="Search for a pool" style="width: 250px" class="mb-6" />
     <a-row :gutter="[16, 16]">
       <a-col v-for="(pool, poolKey) in filteredPools" :key="poolKey" :span="16" :lg="{ span: 12 }" :xl="{ span: 8 }">
-        <div class="flex items-center">
+        <div class="span items-center">
           <a-typography-title :level="5" style="margin: 0">
-            <span class="font-normal mr-4"> {{ poolKey }}: </span>
+            <span class="font-normal mr-4"><b>{{ poolKey }}:</b></span>
           </a-typography-title>
-          <a-input-number
-            v-model:value="pool.value"
-            min="0"
-            max="100"
-            step="1"
-            @change="(val) => onPoolChange(poolKey, val)"
-          />
         </div>
         <div>
-          <!-- <code> {{ pool.description }} </code> -->
+            <!-- <code> {{ pool.description }} </code> -->
         </div>
+        <a-input-number
+          v-model="pool.value"
```

Comment:
What warning were you receiving for this? I'm not getting any warnings- and this doesn't seem to work, the values from the store aren't reflected in the input and the store isn't being updated when changing the inputs. `v-model:value` is [how Ant Design Vue shows it's usage](https://antdv.com/components/input-number).

Reply:
https://eslint.vuejs.org/rules/no-v-model-argument.html

Reply:
If this is not working then I will revert it back to the original state.